requirements.txt

```diff
@@ -1,3 +1,4 @@
 requests==2.31.0
 datasets==2.14.5
 Pillow==10.0.1
+huggingface_hub
```

Comment:
This is needed for the [HFFileSystem](https://huggingface.co/docs/huggingface_hub/v0.18.0.rc0/en/guides/hf_file_system#usage)